src/Symfony/Component/Messenger/Worker.php

```diff
@@ -33,15 +33,15 @@ public function __construct(ReceiverInterface $receiver, MessageBusInterface $bu
     /**
      * Receive the messages and dispatch them to the bus.
      */
-    public function run()
+    public function run(): void
```

Comment:
I wouldn't enforce that now. It might be the case that we need to return something in the future (number of handled messages or something like that) so I'd keep it open to change by not adding a typehint on this extendable class.

Reply:
Removed.